src/device/pf_cmina.h

```diff
@@ -213,6 +213,17 @@ bool pf_cmina_has_timed_out (
    uint16_t period,
    uint16_t factor);
 
+/**
+ * Save the IP settings (and station name) to nonvolatile memory if necessary.
+ *
+ * Compares with the content of already stored settings (in order not to
+ * wear out the flash chip)
+ *
+ * @param net              InOut: The p-net stack instance
+ * @param p_ase            In:    Settings to be saved
+ */
+void pf_cmina_save_ase (pnet_t * net, pf_cmina_dcp_ase_t * p_ase);
```

Comment:
Is this blocking or not? Does it hand off to background task?

Reply:
OK